app/models/async_transaction/vet360/base.rb

```diff
@@ -75,6 +75,7 @@ def self.find_transaction!(user_uuid, transaction_id)
       def self.update_transaction_from_api(transaction_record, api_response)
         transaction_record.status = COMPLETED if FINAL_STATUSES.include? api_response.transaction.status
         transaction_record.transaction_status = api_response.transaction.status
+        transaction_record.metadata = api_response.transaction.messages
```

Comment:
I may be missing something here, but is the encoding and decoding of this field to a JSON-encoded string handled automatically?